config/default/migrate_plus.migration.hs_capx.yml

```diff
@@ -0,0 +1,106 @@
+uuid: 18fe55a6-a1f6-47c9-b079-992ef831c99b
+langcode: en
+status: true
+dependencies: {  }
+id: hs_capx
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: {  }
+migration_group: hs_capx
+label: 'H&S CAPx Importer'
+source:
+  constants:
+    status: 1
+    type: hs_person
+    basic_html: basic_html
+  urls: {  }
+  item_selector: values
+  fields:
+    -
+      name: sunetid
+      label: sunetid
+      selector: uid
+    -
+      name: display_name
+      label: 'Display Name'
+      selector: displayName
+    -
+      name: last_name
+      label: 'Last Name'
+      selector: names/preferred/lastName
+    -
+      name: first_name
+      label: 'First Name'
+      selector: names/preferred/firstName
+    -
+      name: short_title
+      label: 'CAP Short Title'
+      selector: shortTitle/title
+    -
+      name: long_title
+      label: 'CAP Long Title'
+      selector: longTitle/title
+    -
+      name: biosketch
+      label: Biography
+      selector: bio/text
+    -
+      name: email
+      label: Email
+      selector: primaryContact/email
+    -
+      name: alias
+      label: Alias
+      selector: alias
+    -
+      name: profile_photo
+      label: 'Profile Photo'
+      selector: profilePhotos/bigger/url
+    -
+      name: appointments
+      label: 'Administrative Appointments'
+      selector: longTitle/label/text
+    -
+      name: links
+      label: 'Internet Links'
+      selector: internetLinks
+    -
+      name: phone
+      label: 'Phone Numbers'
+      selector: primaryContact/phoneNumbers
+    -
+      name: office
+      label: 'Office Locations'
+      selector: contacts
+    -
+      name: etag
+      label: 'CAP API Etag'
+      selector: meta/etag
+  ids:
+    sunetid:
+      type: string
+process:
+  title: display_name
+  status: constants/status
+  type: constants/type
+  field_hs_person_first_name: first_name
+  field_hs_person_last_name: last_name
+  field_hs_person_title: long_title
+  body/value: biosketch
```

Comment:
biosketch is pulling in the text version of the json response. There is an HTML option available. With the formatting on this field are you sure you want the text version and not the html version?

Reply:
ahh. i didnt realize that. good catch